scan/src/rpc/TxHook.re

```diff
@@ -308,7 +308,8 @@ module Tx = {
     timestamp: MomentRe.Moment.t,
     gasWanted: int,
     gasUsed: int,
-    fee: Coin.t,
+    fee: float,
```

Comment:
Why we change type of fee from coin to float?

Reply:
done